### DISCUSSION TITLE
What is the ASY mode for and what is the hexagon button for?

### DISCUSSION THREAD

**Discussion by duebelpapst** [kachurovskiy/nanoels#149] [General]
Hello,

I am currently playing around with the H4 and haven't found an explanation for these two questions.

Kind regards,
Stefan

**Comment by kachurovskiy** [comment]
Hi @duebelpapst, `ASY` mode moves the Z axis at a constant speed of `Pitch` per second - e.g. if you choose `2mm` pitch then when you switch to `ASY` mode and turn it `ON`, axis will move without the spindle turning. It will still respect the soft limits.

This can be useful e.g. for grinding operations.

Hexagon button was supposed to be used for adjusting settings on device - without updating the firmware - but there currently aren't any such settings yet. They might be added in the future or you can program this button to do something custom by changing the source code.

Cheers!
